opencensus/trace/propagation/jaeger_format.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import re
+
+from opencensus.trace.span_context import SpanContext
+from opencensus.trace.trace_options import TraceOptions
+
+_UBER_HEADER_NAME = 'UBER_TRACE_ID'
```

Comment:
is jaeger literally still using the name uber in its header format? if not, and this feature is desirable in core codebase... maybe parameterize it?

Reply:
It is, but it wont be changing. They intend to move to the w3c trace header spec which is entirely different to the existing header.